.assistance/Vagrantfile

```diff
@@ -45,9 +45,9 @@ Vagrant.configure("2") do |config|
     echo "export GOLANG_VERSION=1.14" >> /home/vagrant/.env
     echo "export RVM_VERSION=1.29.12" >> /home/vagrant/.env
     echo "export RUBY_VERSION=2.4.2" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_USER=user" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_PASSWORD=ZPxcNe6dXZmGmCz3" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_URL=artifactory.stackstate.io/artifactory/api/pypi/pypi-local" >> /home/vagrant/.env
+    echo "export artifactory_user=user" >> /home/vagrant/.env
+    echo "export artifactory_password=ZPxcNe6dXZmGmCz3" >> /home/vagrant/.env
+    echo "export ARTIFACTORY_PYPI_URL=artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-local" >> /home/vagrant/.env
```

Comment:
This should be `artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-virtual`